src/builtin_exit.c

```diff
@@ -0,0 +1,9 @@
+#include "bash_ops.h"
+#include <stdlib.h>
+
+int	builtin_exit(t_command *cmd)
+{
+	(void)cmd;
+	exit(EXIT_SUCCESS);
```

Comment:
as they say, RTFM -> do 'man exit'

check the behaviour of exit by running it in a subshell (exit), (exit 1), (exit a as das ) etc

convert string @ argv[1] to unsigned char integer using libft function

if not a string @ argv[1], just exit 0
if more than one argument, return 1 (and put a comment so that we put an error output here later)